app/frontend/src/features/contributorForm/ContributorForm.tsx

```diff
@@ -0,0 +1,384 @@
+                          {errors?.contribute?.message ?? " "}
+                        </FormHelperText>
+                      </FormControl>
+                    )}
+                  />
+                  <p>{QUESTIONS_OPTIONAL}</p>
+                  <TextField
+                    inputRef={register}
+                    id="ideas"
+                    margin="normal"
+                    name="ideas"
+                    label={IDEAS_LABEL}
+                    helperText={IDEAS_HELPER}
+                    fullWidth
+                    multiline
+                    rows={4}
+                    rowsMax={6}
+                  />
+                  <TextField
+                    inputRef={register}
+                    id="features"
+                    margin="normal"
+                    name="features"
+                    label={FEATURES_LABEL}
+                    helperText={FEATURES_HELPER}
+                    fullWidth
+                    multiline
+                    rows={4}
+                    rowsMax={6}
+                  />
+                  {!authState.authenticated && (
+                    <>
+                      <TextField
+                        inputRef={register}
+                        id="age"
+                        type="number"
+                        margin="normal"
+                        name="age"
+                        label={AGE}
+                        fullWidth
+                      />
+                      <Controller
+                        id="gender"
+                        control={control}
+                        name="gender"
+                        render={({ onChange }) => (
+                          <FormControl>
+                            <FormLabel component="legend">{GENDER}</FormLabel>
+                            <RadioGroup
+                              className={classes.genderRadio}
+                              aria-label="gender"
+                              name="gender-radio"
+                              onChange={onChange}
+                            >
+                              {GENDER_OPTIONS.map((opt) => (
+                                <FormControlLabel
+                                  key={opt}
+                                  value={opt}
+                                  control={<Radio />}
+                                  label={opt}
+                                />
+                              ))}
+                            </RadioGroup>
+                          </FormControl>
+                        )}
+                      />
+                      <TextField
+                        inputRef={register}
+                        id="location"
+                        margin="normal"
+                        name="location"
+                        label={LOCATION_LABEL}
+                        helperText={LOCATION_HELPER}
+                        fullWidth
+                      />
```

Comment:
```suggestion
                      <TextField
                        inputRef={register}
                        id="location"
                        margin="normal"
                        name="location"
                        label={LOCATION_LABEL}
                        helperText={LOCATION_HELPER}
                        fullWidth
                      />
```
Set the `defaultValue` prop on this (got a react-hook-form warning on this) with the result of `getContributorFormInfo`, should be accessible if using react query to do this once loaded.